tcks/microprofile-health/tck-suite.xml

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
```

Comment:
Do we need the file if it doesn't change anything? SmallRye Health currently doesn't have this file.

Is it purely a convenience for if we need to exclude something in the future?

Reply:
I'd prefer having this file so that it's clear that we don't exclude anything.